Extension/package.json

```diff
@@ -6573,6 +6573,7 @@
         "xml2js": "^0.6.2"
     },
     "dependencies": {
+        "@github/copilot-language-server": "^1.253.0",
```

Comment:
@lukka There's a 1.254.0 currently -- should we switch to that one instead. Do you know how important it is to update this? I'm guessing it might be important since it's so new?

Reply:
Also, unlike most packages, there appears to be no publicly available changelog so I can't tell what's different.